service-directory/cluster-ip-service/README.md

```diff
@@ -0,0 +1,131 @@
+# Service Directory GKE Integration - ClusterIP Service
+
+Service Directory for GKE is a cloud-hosted controller for GKE Clusters that
+sync services to Service Directory.
```

Comment:
Can you please capitalize the S in service to make sure it points to Kubernetes Services.

Reply:
I've fixed all of the capitalization issues in the README, and added a main README for the folder that points to all of the examples.